interface/wx/timectrl.h

```diff
@@ -150,11 +150,11 @@ class wxTimePickerCtrl : public wxControl
         The date part of @a dt is ignored, only the time part is displayed in
         the control. The @a dt object must however be valid.
 
-        In particular notice that it is a bad idea to use default wxDateTime
+        In particular, notice that it is a bad idea to use default wxDateTime
         constructor from hour, minute and second values as it uses the today
-        date for the date part which means that some times can be invalid if
+        date for the date part, which means that sometimes can be invalid if
```

Comment:
Sorry, but this is still wrong, "some times" here means "some wxDateTime objects".

Please do not change things when in doubt, it's better to leave a possible typo than "fix" it and change the meaning of the sentence.